core/api/test/legacy-integration/app/trigger/trigger.fn.spec.ts

```diff
@@ -104,6 +122,9 @@ describe("onchainBlockEventHandler", () => {
     let lastHeight = initialBlock
     const subBlocks = subscribeToBlocks({ lnd: lnd1 })
     subBlocks.on("block", async ({ height }: { height: number }) => {
+      // this sleep seems necessary to allow confirmations to propagate for
+      // 'getChainTransactions' call inside 'listIncomingOnChainTransactions'
+      await sleep(500)
```

Comment:
then is something else.... please dont use this inside the handler

Reply:
Ok removed here, will troubleshoot again when we're actually removing `01` folder https://github.com/GaloyMoney/galoy/pull/3677/commits/be614f537fe31258b505c317222add42acdb2f11

Reply:
Oh actually, we already have this test in bats #3710